ui/app/AppLayouts/Wallet/panels/WalletHeader.qml

```diff
@@ -74,6 +78,45 @@ Item {
             Layout.alignment: Qt.AlignTrailing
             Layout.topMargin: 5
 
+            Row {
+                Layout.alignment: Qt.AlignRight | Qt.AlignTop
+                Layout.preferredHeight: 38
+                spacing: 8
+
+                StatusButton {
+                    id: reloadButton
```

Comment:
After click on this button we should disable it for at least one minute, to avoid users constantly clicking on it.